packages/plugin-core/src/commands/CopyNoteLink.ts

```diff
@@ -37,8 +45,52 @@ export class CopyNoteLinkCommand extends BasicCommand<
     window.showInformationMessage(`${link} copied`);
   }
 
+  private async getUserLinkAnchorPreference(): Promise<
+    "line" | "block" | null
+  > {
+    const { config, wsRoot } = ExtensionProvider.getDWorkspace();
+    let anchorType = ConfigUtils.getNonNoteLinkAnchorType(config);
+    if (anchorType === undefined) {
+      // The preferred anchor type is not set, so ask the user if they want line numbers or block anchors
```

Comment:
makes me think if this should be a third option (eg. prompt) when anchor type is not set.